### PR TITLE
Add function to receive token.

### PR DESCRIPTION
**Pull request by galachad**
Improve readme.
Add function to receive/renovate a token.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -57,3 +58,41 @@ modify_vector <- function(x, y = NULL) {
...
+
+  parsed <- jsonlite::fromJSON(httr::content(resp, "text", encoding = "UTF-8"), simplifyVector = FALSE)
+
+  if (httr::http_error(resp)) {
+    stop(glue("Auth0 API request failed [{httr::status_code(resp)}]\n{parsed$error_description}"), call. = FALSE)
+  }
+
+  set_token(parsed$access_token)
+  message(glue("The new token was set in the environment."))
+  browser()
```
Comment (tomecki) [inline R/utils.R]: browser() statement

File: R/utils.R [hunk 2]
```diff
@@ -57,3 +58,41 @@ modify_vector <- function(x, y = NULL) {
...
+#' This function allows to generate new auth token base on client id and secret
+#' If you looking for more infromation see: \url{https://auth0.com/docs/api/management/v2/tokens}
+#' @param client_id The OAuth application id
+#' @param client_secret The OAuth application secret
+#' @param .domain By default \code{\link{api_domain}}
+#' @return Auth0 Bearer token
+#' @export
+generate_token <- function(client_id, client_secret, .domain = api_domain()) {
+  resp <-
+    httr::POST(
```
Comment (tomecki) [inline R/utils.R:73]: what happens on failed request (like connectivity issue?) Are we handling that?

**Review by tomecki — CHANGES_REQUESTED** [review]
Minor comments

File: R/utils.R (hunk 1 above)
Comment (galachad) [reply R/utils.R]: Thanks for catch this. That was just a debugged line that I forget to remove.

File: R/utils.R (hunk 2 above)
Comment (galachad) [reply R/utils.R:73]: This is the correct behavior. The function just returns an error and stops the execution.

File: DESCRIPTION [hunk 3]
```diff
@@ -15,7 +15,8 @@ Depends:
     purrr
 Imports:
     utils,
-    grDevices,
-    httr
```
Comment (tomecki) [inline DESCRIPTION:19]: it still imports httr, right?

**Review by tomecki — CHANGES_REQUESTED** [review]
final question

**Review by krystian8207 — APPROVED** [review]
lgtm

File: DESCRIPTION (hunk 3 above)
Comment (galachad) [reply DESCRIPTION:19]: We import all needed functions in the namespace of the package also `httr` is in dependency of the package.

**Pull request merged**